python/tk_unreal/unreal_sg_engine.py

```diff
@@ -162,25 +163,26 @@ def _execute_within_exception_trap(self):
                 self._callback()
             except Exception as e:
                 current_engine = sgtk.platform.current_engine()
+                current_engine.logger.debug("%s" % e, exc_info=True)
                 current_engine.logger.exception("An exception was raised from Toolkit")
             self._callback = None
-                
+
     @unreal.ufunction(override=True)
     def shutdown(self):
         from sgtk.platform.qt5 import QtWidgets
```

Comment:
Is ok to make explicit import ?

Reply:
I don't know if this was done on purpose or not, and I don't know the potential side effects of changing it.